integration-tests/maven/src/test/java/io/quarkus/maven/it/LaunchMojoTestBase.java

```diff
@@ -34,7 +34,8 @@ public void testThatTheTestsAreReRunMultiModule()
         ContinuousTestingMavenTestUtils.TestStatus results = testingTestUtils.waitForNextCompletion();
 
         //check that the tests in both modules run
-        assertEquals(2, results.getTestsPassed());
+        assertEquals(2, results.getTestsPassed(),
+                "Did not meet expectation for number of tests passed, actual results " + results);
```

Comment:
I think we should really try to use AssertJ more broadly. It provides a much  better output when assertions are false.

Probably a good idea to use it if you have other adjustments to make.